doc/source/ray-air/examples/analyze_tuning_results.ipynb

```diff
@@ -0,0 +1,918 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "c62191af",
+   "metadata": {},
+   "source": [
+    "# Analyzing results from hyperparameter tuning\n",
+    "In this example we'll run a hyperparameter tuning experiment and analyze the results.\n",
```

Comment:
Nit: maybe add a bit more description here.

In this example, we will go through how you can use Ray AIR to run a distributed hyperparameter experiment to find optimal hyperparameters for an XGBoost model.